eslint.config.js

```diff
@@ -88,6 +88,12 @@ module.exports = {
       parserOptions: {
         sourceType: 'module',
       }
+    },
+    {
+      files: ["test/**/*.js", "test/**/*.mjs"],
+      rules: {
+        "no-unused-vars": "off",
+      }
```

Comment:
Reviewer note: We turn on the same rule for .ts files in test/... dirs.
See https://github.com/open-telemetry/opentelemetry-js-contrib/pull/2940#issuecomment-3080842540 for discussion.